front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -411,11 +414,18 @@ export function MessageView({
               </div>
             ) : (
               <div className="flex">
-                <img
-                  className="h-8 w-8 rounded-md"
-                  src={user?.image || "https://gravatar.com/avatar/anonymous"}
-                  alt=""
-                />
+                {!readOnly && user?.image ? (
+                  <img
+                    className="h-8 w-8 rounded-md"
+                    src={
+                      user?.image ||
+                      "https://gravatar.com/avatar/anonymous?d=mp"
+                    }
+                    alt=""
+                  />
+                ) : (
+                  <UserCircleIcon className="mx-1 my-1 h-6 w-6 text-gray-300"></UserCircleIcon>
```

Comment:
no case in which gravatar url will show anymore -- just checking it's indeed the intended behavior

Reply:
Will fix

Reply:
(done)